osu.Game/Overlays/Comments/TotalCommentsCounter.cs

```diff
@@ -33,7 +34,7 @@ public TotalCommentsCounter()
                 Spacing = new Vector2(5, 0),
                 Children = new Drawable[]
                 {
-                    new OsuSpriteText
+                    text = new OsuSpriteText
```

Comment:
Is there a reason for keeping all this drawable init in the ctor, rather than moving to bdl and assigning colours directly?